### ISSUE TITLE
Adicionar uma nova página para portagens "Pitch"

### ISSUE THREAD

**Issue by mthmcalixto** [filipedeschamps/tabnews.com.br#1804]
### Descrição

A ideia seria criar algo exclusivo para postagens "Pitch", remover elas do "relevantes e recentes" e deixar apenas em "Pitch".

**Comment by Rafatcb** [comment]
Vi algumas pessoas reclamando que tem muitos `Pitch`s, e que gostariam de ver menos. A implementação de um conteúdo `Pitch` já está meio encaminhada no código/banco, mas criar uma aba não irá resolver as reclamações que citei. O mesmo vale para as reclamações sobre a quantidade de publicações da `NewsletterOficial`. Digo isso porque todas publicações continuariam aparecendo em `Recentes` (`Publicações` e `Todos`).

Não sei se essa foi a motivação para sua sugestão, mas acho importante ter isso em mente para quando desenvolvermos uma implementação.

Um filtro seria bom, e tem várias sugestões que parecem demandar isso. Hoje, as listas de conteúdo são estáticas.

**Comment by mthmcalixto** [comment]
> Vi algumas pessoas reclamando que tem muitos `Pitch`s, e que gostariam de ver menos. A implementação de um conteúdo `Pitch` já está meio encaminhada no código/banco, mas criar uma aba não irá resolver as reclamações que citei. O mesmo vale para as reclamações sobre a quantidade de publicações da `NewsletterOficial`. Digo isso porque todas publicações continuariam aparecendo em `Recentes` (`Publicações` e `Todos`).
> 
> Não sei se essa foi a motivação para sua sugestão, mas acho importante ter isso em mente para quando desenvolvermos uma implementação.
> 
> Um filtro seria bom, e tem várias sugestões que parecem demandar isso. Hoje, as listas de conteúdo são estáticas.

Exatamente, está muito pitch nos relevantes, mas a ideia seria criar uma aba mesmo e deixar apenas os tops lá, eles vão aparecer em recentes normal, mas quando for subir pro topo, vão pra aba em vez de ir para relevantes.

**Comment by gpoleszuk** [comment]
Boa ideia! Desta forma separa o que são páginas amarelas e o que é conteúdo relevante.

A ideia de criar uma nova aba ainda não parecer ser um problema, mas no futuro ter aba para diferentes tópicos que vão surgir, vai carregar a interface. Não sei se é possível trabalhar com _tags_ (palavras chaves) definidas para cada postagem pelo próprio criador. Com isso, elas ajudariam a filtrar as publicação de um momento em diante. Posso até atualizar as publicações que já fiz, pois são poucas.

Sou apenas um usuário do Tabnews e não imagino o trabalhão dos desenvolvedores para implementar essas diferentes ideias que aparecem. Parabéns por disponibilizarem este espaço para pelo menos avaliarem se são factíveis, podem aguardar ou mesmo impossíveis :).

**Comment by samsepiol1** [comment]
Isso é algo que percebo faz um tempo, na verdade, enxergo como um fluxo natural de uma plataforma social e falo até mesmo pelo contexto do social e economico do país: 

# É inevitável? Isso é um fluxo natural?
Sim, tende a ser um fluxo natural, principalmente em plataformas que se expandem rapidamente e atraem muitos usuários novos. As etapas desse processo são normalmente assim:

- Fase Inicial – Conteúdo genuíno e focado em agregar valor (comunidade pequena e engajada).

- Fase de Crescimento – A plataforma começa a atrair novos usuários com diferentes interesses e objetivos.

- Fase de Saturação – As pessoas começam a competir por visibilidade e audiência. Monetização vira uma prioridade para muitos.

- Fase de Curadoria (ou Declínio) – A comunidade ou a plataforma precisa se adaptar e implementar formas de manter qualidade. Caso contrário, usuários insatisfeitos começam a abandonar o espaço.

Sobre o contexto socio economico que falei anteriormente: Com o crescimento de fenômenos como influenciadores digitais e startups, muitos enxergam a internet como uma oportunidade de negócio.

Histórias de sucesso viral (por exemplo, startups SaaS ou projetos que "bombam" rapidamente) criam um ciclo de imitação: as pessoas tentam reproduzir modelos lucrativos sem inovar, esperando resultados semelhantes. E como a questão do *marketing digital é bem forte aqui* acaba que as pessoas percebem o tabnews como uma plataforma de promoção do que como algo colaborativo. 

Em adiação a isso penso o seguinte: Muitas pessoas enxergam essas plataformas como canais diretos de vendas e começam a usar o espaço para divulgar serviços e produtos, muitas vezes sem agregar conteúdo relevante.

Há, certamente, uma forte cultura de consumo de cursos e infoprodutos, com pitches e promessas de "fórmulas prontas" para ganhar dinheiro online, o que incentiva ainda mais essas práticas. Falo isso não só do tabnews, mas basta oberservar em outras redes sociais também como Tik Tok e instagram. 

# Mecanismo de Incentivo e Economia de Atenção

Existe alguns mecanismos ou falta de mecanismo que corroboram para isso também. Sei que o desenvolvimento é bem complexo, acompanho algumas issues no repositório oficial da plataforma. Acredito que a falta de um algoritmo ou categorias das postagens pesa um pouco também. tem muita gente que gosta de ver conteúdos mais tećnicos e gente que gosta de ver ideias ou que tá sendo produzido. Vai de cada um, para mim, uma solução de médio a longo prazo seria a *personalização*


# Como a Personalização e os Algoritmos de Categoria Ajudam?

A necessidade de personalização surge em plataformas quando o volume de conteúdo e a diversidade de interesses dos usuários começam a crescer além do que é possível organizar de forma manual ou linear. Abaixo, detalho as fases típicas de um fluxo natural de uma plataforma e em qual momento a personalização se torna essencial para manter a relevância e o engajamento. Para mim duas abordagens seriam bem interessantes para o TabNews: (1) Categoriazação com Tags Inteligentes e (2) Comunidades dentro do Tabnews


1. Categorização Avançada com Tags Inteligentes

- A categorização de conteúdo ajuda a organizar postagens em tópicos específicos (como Marketing, Tecnologia, Relacionamento, SaaS), permitindo que os usuários filtrem o que querem ver.

- Algoritmos de categorização automática podem analisar o conteúdo e aplicar tags apropriadas, evitando que posts de marketing apareçam em categorias técnicas, por exemplo.


2. Criação de Comunidades e Subgrupos Personalizados
- Algoritmos podem facilitar a criação de subcomunidades dentro da plataforma, agrupando usuários com interesses similares.

- Esses subgrupos permitem que pessoas compartilhem conteúdo mais específico, sem poluir o feed principal da plataforma.

**Comment by thonycsdev** [comment]
Ando companhando o site desde o lançamento e tem sido algo que me incomoda também. Vi alguns posts que parecem vendas de SaaS para gerar um público, não concordo com esse tipo de conteúdo fora da aba de AD. 

Minha sugestão pessoal seria uma aba para buscar contribuidores open-source onde o usuário faz seu pitch sobre uma ideia. A comunidade do TabNews/Open-Source podem contribuir com a ideia, podendo gerar até melhores resultados do que foi inicialmente previsto.

**Comment by ItaloSa** [comment]
Acho muito improvável algo desse tipo entrar. A ideia é que o Tabnews siga mais ou menos o que é o Hacker News

**Comment by DarleiC** [comment]
Vou cutucar nessa issue parada aqui a tanto tempo para contribuir com algo que possa ajudar no assunto. 

Eu criei uma extensão que tenta minimizar esse "problema" do spam de PITCH.

Pode não ser a solução definitiva, mas talvez pode servir de inspiração para o site pensar em algo nessa linha.
O que a extensão deve fazer é mostrar como se já estivesse lido, assim a gente só ignora e olha para as outras postagens.

[Post no TabNews sobre a extensão](https://www.tabnews.com.br/DarleiC/madrugada-marcada-como-lida)
